spec/features/idv/doc_auth/how_to_verify_spec.rb

```diff
@@ -72,4 +72,199 @@
       expect(page).to have_current_path(idv_hybrid_handoff_url)
     end
   end
+
+  describe 'navigating to How To Verify from Agreement page in 50/50 state' do
+    before do
+      allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+    end
+
+    context 'opt in false at start but true during navigation' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { false }
+
+        sign_in_and_2fa_user
+        complete_doc_auth_steps_before_agreement_step
+        complete_agreement_step
+      end
+
+      it 'should not be bounced back from Hybrid Handoff to How to Verify' do
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+        page.refresh
+        expect(page).not_to have_current_path(idv_how_to_verify_url)
```

Comment:
maybe we dont need this expect considering the one that follows this

Reply:
Yep! Good catch, removing.